packages/ui-react/src/pages/ScreenRouting/mediaScreens/MediaLiveChannel/MediaLiveChannel.tsx

```diff
@@ -1,24 +1,25 @@
 import { Navigate } from 'react-router-dom';
 import type { PlaylistItem } from '@jwp/ott-common/types/playlist';
-import { isLiveChannel } from '@jwp/ott-common/src/utils/media';
 import { liveChannelsURL } from '@jwp/ott-common/src/utils/urlFormatting';
 
 import type { ScreenComponent } from '../../../../../types/screens';
-import ErrorPage from '../../../../components/ErrorPage/ErrorPage';
 import Loading from '../../../Loading/Loading';
+import MediaMovie from '../MediaMovie/MediaMovie';
 
 const MediaLiveChannel: ScreenComponent<PlaylistItem> = ({ data, isLoading }) => {
-  const liveChannelsId = isLiveChannel(data) ? data.liveChannelsId : undefined;
+  const liveChannelsId = data.liveChannelsId;
 
-  if (data && !isLoading && liveChannelsId) {
-    return <Navigate to={liveChannelsURL(liveChannelsId, data.mediaid)} replace={true}></Navigate>;
+  if (isLoading) {
+    return <Loading />;
   }
 
-  if (!liveChannelsId) {
-    return <ErrorPage title="Live channel not found" />;
+  // this live channel is part of a live channels (TV Guide) page
+  if (liveChannelsId) {
+    return <Navigate to={liveChannelsURL(liveChannelsId, data.mediaid)} replace={true}></Navigate>;
   }
 
-  return <Loading />;
+  // this live channel is "just" a media item
+  return <MediaMovie data={data} isLoading={isLoading} />;
```

Comment:
This PR fixes the "Live channel not found" when creating a live stream via JWP dashboard. 

We could improve it, but we might need to agree to some content-type standards first. Because `liveChannel` was intended to be used for live streams with EPG and live channels page. Since this will take some time, I suggest that we first ensure live streams are working again 😄